draft-ietf-quic-multipath.md

```diff
@@ -237,6 +248,16 @@ defined as follows:
   the multipath extension as defined in this document. This parameter has
   a zero-length value.
 
+- max_concurrent_paths (current version uses 0xced74c7a): This is 
```

Comment:
Why not make this the value of `enable_multipath`? You only need a single transport parameter then.

Reply:
Yes, we could use the same transport parameter.

Reply:
I prefer change it later as people may still need to be compatible to their implementations. We can combine these 2 transport parameters after this PR is agreed to be merged by the working group.

Reply:
Up to you. Since the mechanism described in this PR is significantly different from the current draft version, you could also bump the code point of `enable_multipath`.

Reply:
Updated that.

Reply:
I also update the definition for these transport parameter, combined 2 TPs and replace the old "enable_multipath" with "initial_max_paths". 
Code point of -07 would be able for implementations to be compatible.